mergify_engine/tests/functional/actions/test_merge.py

```diff
@@ -280,6 +280,49 @@ def test_merge_github_workflow(self):
         assert check["conclusion"] == "cancelled"
         assert check["output"]["title"] == "The rule doesn't match anymore"
 
+    def test_merge_depends_on(self):
```

Comment:
I guess this does not work accross repos. I mean, you don't get an event in this case so it needs to be refreshed manually?